docker/build-tool/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:buster
+COPY ./dev_setup.sh /build_setup.sh
+COPY ./rust-toolchain.toml /rust-toolchain.toml
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update -y && \
```

Comment:
Maybe it's better to set `ENV DEBIAN_FRONTEND=noninteractiv`?

Reply:
BTW, I believe the value should be `noninteractive`. @ZhiHanZ 

ref: https://www.debian.org/releases/sarge/s390/ch05s02.html.en